### DISCUSSION TITLE
[#1040938] Going through blocks in reticle mode

### DISCUSSION THREAD

**Discussion by RRRoriginal** [Mojang/minecraft-editor#108] [Bugs]
**Repro steps**
1. Activate reticle (crosshair) mode
2. Start to fly
3. Touch a block

**Observed result vs. what I actually expected to happen**
When the character touches a block, it goes through. This also happens when flying up or down. When it goes through a block, it stops flying. I think it should be able to go through blocks comfortably or just do not be able to do it.

**Other helpful info**

**Preview build version:** 1.20.0.21
**Frequency:** Always. It stops flying most of the times.
**Local single-player, peer-to-peer, dedicated server (and how many people):** Single-player
**Graphics card:** DirectX12
**Device info:** Laptop Asus

**Comment by xKingDark** [comment]
**Observed result**:
https://user-images.githubusercontent.com/47607185/234439952-d2e7acf6-7d28-48b1-824e-b8d0de00fa1c.mov
https://user-images.githubusercontent.com/47607185/234440507-a7252bee-42b8-461a-9bcf-924aeea4fe15.mov

**Comment by m-brisch** [comment]
Hey @RRRoriginal, this should be resolved now!
